asdf/tests/test_versioning.py

```diff
@@ -278,12 +278,9 @@ def xfail_version_map_support_cases(request):
     version = request.getfixturevalue("version")
     if (version, tag) in [
         ("1.6.0", "tag:stsci.edu:asdf/core/column-1.1.0"),
-        ("1.6.0", "tag:stsci.edu:asdf/core/integer-1.1.0"),
-        ("1.6.0", "tag:stsci.edu:asdf/core/ndarray-1.1.0"),
         ("1.6.0", "tag:stsci.edu:asdf/core/table-1.1.0"),
         ("1.6.0", "tag:stsci.edu:asdf/fits/fits-1.1.0"),
         ("1.6.0", "tag:stsci.edu:asdf/time/time-1.2.0"),
-        ("1.6.0", "tag:stsci.edu:asdf/unit/defunit-1.0.0"),
         ("1.6.0", "tag:stsci.edu:asdf/unit/defunit-1.1.0"),
         ("1.6.0", "tag:stsci.edu:asdf/unit/quantity-1.2.0"),
         ("1.6.0", "tag:stsci.edu:asdf/unit/unit-1.1.0"),
```

Comment:
I removed some of the skips added in #1251 and #1252
Removing the remaining skips would require updates to astropy.io.misc.asdf or merging the PRs in asdf-standard that remove these tags (table, column, time, unit, etc) from the standard version maps.